src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -42,6 +40,8 @@
     <!-- These prevent the default MsBuild targets from referencing System.dll and mscorlib.dll -->
     <ExcludeMscorlibFacade>true</ExcludeMscorlibFacade>
     <RuntimeMetadataVersion>v4.0.30319</RuntimeMetadataVersion>
+    <!-- Override InformationalVersion during servicing as it's returned via public api. -->
+    <InformationalVersion Condition="'$(PreReleaseVersionLabel)' == 'servicing'">$(ProductVersion)</InformationalVersion>
```

Comment:
@safern, I'm pinning the `InformationalVersion` for both CoreLibs in servicing to avoid the reported issue. Pinning `InformationalVersion` directly instead of relying on `Version` seems to be the better choice as it means properties like `IsShipping` or others won't interfere.

I checked the InformationalVersion for 5.0 P8 which is `5.0.0-preview.8.20407.11` which makes sense and means that it should only be pinned during servicing.

Please let me know if that change makes sense to you. For more context, please find this perfect blogpost from Andrew Lock: https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/